INSTALL.rst

```diff
@@ -11,11 +11,12 @@ Installation
 ============
 
 This package can be checked out from
-svn://svn.zope.org/repos/main/z3c.saconfig/trunk. You can then execute
+https://github.com/zopefoundation/z3c.saconfig. You can then execute
 the buildout to download and install the requirements and install the
 test runner. Using your desired python run::
 
-  $ python bootstrap.py
+  $ python3 -m venv .
+  $ bin/pip install -r requirements.txt
```

Comment:
This is missing a step:

    $ bin/buildout

(since setuptools and zc.buildout are the only packages mentioned in requirements.txt, I assume zc.buildout is still meant to be used.)

Reply:
This step is in line 23 the first one outside the diff shown on GitHub.